src/linpred.jl

```diff
@@ -294,50 +385,100 @@ function modelframe(f::FormulaTerm, data, contrasts::AbstractDict, ::Type{M}) wh
     f, modelcols(f, data)
 end
 
-modelmatrix(obj::LinPredModel) = obj.pp.X
+modelframe(obj::LinPredModel) = obj.fr
+
```

Comment:
This method signature isn't used nor documented anywhere AFAIK, and `fr` doesn't exist.
```suggestion
```